.github/workflows/dotcom-acceptance-tests-all.yml

```diff
@@ -8,7 +8,7 @@ on:
     #         │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
     #         │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
     #         * * * * *
-    - cron:  '0 0 * * *'
+    - cron:  '0 0 * * 3'
```

Comment:
Is the goal on this change to save on rate limits? Overall GitHub resources/churn?

Reply:
Yes!